Library/Homebrew/test/testing_env.rb

```diff
@@ -20,6 +20,25 @@
   abort "Run `bundle install` or install the mocha and minitest gems before running the tests"
 end
 
+if ENV["HOMEBREW_TESTS_COVERAGE"]
+  # Ensure all Ruby files are `require`d for an accurate coverage measurement
+  Dir[File.expand_path("../../**/*.rb", __FILE__)].each do |f|
+    next if [
+      # scripts
+      "build.rb", "postinstall.rb", "test.rb",
+      # mocked files
+      "config.rb", "global.rb",
+    ].any? do |core_file|
+      f.end_with?("Homebrew/#{core_file}")
+    end
+
+    # ignore test files
+    next if f.include?("/test/") || f.include?("/vendor/bundle/") || f.end_with?("cmd/test.rb")
```

Comment:
The last one in this list should have been `cmd/tests.rb`? Why do we need an extra check for `/vendor/bundle/`? Isn't that a subdirectory of `test` and already covered by `/test/`?